src/telliot_feeds/reporters/tips/listener/funded_feeds_filter.py

```diff
@@ -124,6 +124,9 @@ def filter_historical_submissions(self, feeds: list[QueryIdandFeedDetails]) -> l
             for current, previous in zip(
                 feed.queryid_timestamps_values_list[::-1], feed.queryid_timestamps_values_list[-2::-1]
             ):
+                if feed.params.startTime > current.timestamp:
```

Comment:
What's this doing?

Reply:
if current timestamp is before feed start then no need to check